cuda_bindings/cuda/bindings/_lib/param_packer.pxd

```diff
@@ -1,5 +1,7 @@
 # SPDX-FileCopyrightText: Copyright (c) 2021-2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 # SPDX-License-Identifier: LicenseRef-NVIDIA-SOFTWARE-LICENSE
 
-cdef extern from "param_packer.h":
+# Include "param_packer.cpp" so its contents get compiled into every
+# Cython extension module that depends on param_packer.pxd.
+cdef extern from "param_packer.cpp":
```

Comment:
If we aren't compiling `param_packer.cpp` anymore, it feels a bit weird to keep it as a `.cpp` file. Maybe put it all in `param_packer.h` and extern it the same way here? That would at least make it a bit clearer from the cpp side how it's being used?